src/app/account/invitation/[inviteId]/page.tsx

```diff
@@ -11,18 +10,16 @@ export default async function AcceptInvitePage({ params }: { params: Promise<{ i
 
     const invite = await db
         .selectFrom('pendingUser')
-        .select('email')
+        .select(['email', 'orgId']) // Fetch orgId to potentially get orgName later if needed
         .where('claimedByUserId', 'is', null)
         .where('id', '=', inviteId)
         .executeTakeFirst()
 
-    if (!invite) return <ErrorAlert error="Invalid invitation" />
+    if (!invite) return <ErrorAlert error="Invalid or already claimed invitation." />
 
     return (
         <Container>
-            <Panel title="Welcome To SafeInsights">
-                <AccountPanel inviteId={inviteId} email={invite.email} />
-            </Panel>
+            <InvitationHandler inviteId={inviteId} invitedEmail={invite.email} />
```

Comment:
why did you add orgId if you didn't pass it on here:
```suggestion
            <InvitationHandler invite={invite} />
```

Reply:
this has been an oversight from an old approach. I've removed it now.